packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,22 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessible label for the checkbox. Defaults to 'label' property value.
+   */
+  ariaLabel?: string;
+
+  /**
+   * ID for element that contains label information for the checkbox.
+   * Defaults to 'label' property value.
```

Comment:
@phkuo, I see your point but as I said `htmlFor` automatically makes sure that label for control is assigned for accessibility purpose. I really want to be descriptive to users to avoid confusion here. How about : "By default, label prop provides the label information" ? Does that sound okay ?